changelog.txt

```diff
@@ -1,3 +1,17 @@
+Hector 2.0.0
+============
+* Incorporated 1-D diffusive ocean heat model (DOECLIM)
```

Comment:
Be consistent with capitalization: either don't capitalize this line, or capitalize the following ones.